src/components/plots/scatterPlot.ts

```diff
@@ -23,20 +23,14 @@ export module Plots {
                                            .delay(5));
       this.attr("opacity", 0.6);
       this.attr("fill", new Scales.Color().range()[0]);
+      this.size(6);
+      this.symbol(() => SymbolFactories.circle());
```

Comment:
Does this create a new `SymbolFactory` each time it's invoked? If so, you might consider doing:
```Typescript
var circleFactory = SymbolFactories.circle();
this.symbol(() => circleFactory);
```